controllers/grid/users/reviewer/ReviewerGridRow.inc.php

```diff
@@ -34,6 +47,13 @@ function initialize($request, $template = null) {
 		$stageId = (int) $request->getUserVar('stageId');
 		$round = (int) $request->getUserVar('round');
 
+		// Authors can't perform any actions on blind reviews
+		$reviewAssignment = $this->getData();
+		$isAuthorBlind = in_array($reviewAssignment->getReviewMethod(), array(SUBMISSION_REVIEW_METHOD_BLIND, SUBMISSION_REVIEW_METHOD_DOUBLEBLIND));
+		if ($this->_isCurrentUserAssignedAuthor && $isAuthorBlind) {
+			return;
```

Comment:
Should this be a `fatalError` or an `assert(false)`? I think this is probably protected from the grid's `authorize` method already, no?

Reply:
I think we still need this because the row still exists. It just says "Anonymous Reviewer" and it exits early so that no actions are added to the row.

Reply:
Ah, that makes sense.